dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf

```diff
@@ -19,6 +19,11 @@ streamingModelConfig {
     s3BucketName: "mlflow"
   }
 
+  pmml {
+    modelsDirectory: "http://proxy:5100"
+    modelDirectoryHrefSelector: "body > ul > li > a"
```

Comment:
This looks like a hack, we could code up a function which returns a JSON from the server.